base/utf8proc.jl

```diff
@@ -121,6 +121,12 @@ end
 
 charwidth(c::Char) = Int(ccall(:utf8proc_charwidth, Cint, (UInt32,), c))
 
+# faster x+y that does no overflow checking
+p(x::Char, y::UInt32) = reinterpret(Char, reinterpret(UInt32, x) + y)
```

Comment:
Can this please not be a 1-letter function name? I know it's not exported, but still...